float32 x_distance
float32 y_distance
int16   captured
